CONTRIBUTING.md

```diff
@@ -0,0 +1,19 @@
+## Issue Tracker
+> http://github.com/json-api/json-api/issues
```

Comment:
does this actually link in markdown output? also, there probably should be a newline between this and the first line. same on line 13